.env.dist

```
@@ -0,0 +1,128 @@
+_APP_DB_SCHEMA=
+_APP_DB_USER=
+_APP_DNS=
+_APP_DOCKER_HUB_PASSWORD=
+_APP_DOCKER_HUB_USERNAME=
+_APP_DOMAIN_FUNCTIONS=
+_APP_DOMAIN_SITES=
+_APP_DOMAIN_TARGET_A=
+_APP_DOMAIN_TARGET_AAAA=
+_APP_DOMAIN_TARGET_CAA=
+_APP_DOMAIN_TARGET_CNAME=
+_APP_DOMAIN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The _APP_DOMAIN key should go before the _APP_DOMAIN_FUNCTIONS key

(UnorderedKey)